stories/SQFormDialogStepper.stories.js

```diff
@@ -240,6 +242,17 @@ export const SQDialogStepperWithValidation = () => {
             lastName: Yup.string().required('Required')
           }}
         >
+          <SQFormResetInitialValuesButton
+            tooltip="Clear Addresses"
+            confirmationContent="You are about to reset this form. Any unsaved info for this customer will be removed"
+            initialValuesObject={{
```

Comment:
You should create a `personalDateInitialValues` object with the values for that step (firstName: '', lastName: '', middleI: '', alias, newAccount).  Then you can spread that into `initialValuesObject` here and above into the `initialValues` on line 224 so that we don't have mismatched initial values.

Reply:
@Chris-Boyle used them separately as I wanted to show that we can assign new values also If we create one object `personalDateInitialValues` for both` initialValuesObject` and `initialValues ` we won't be able to demonstrate that.

Reply:
What would be a scenario where we would want to do that?  Maybe we have some prefilled values for the initial values.  Then when we hit Reset we would want that step to have no values?  You can have one example doing that.  Any the other one doing what I suggested since I believe what I suggested before is what most consumers would do.